src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const tasks = [
+  {
+    index: 1,
+    description: 'structure todo list',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'add task todo list',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'remove todo list',
+    completed: false,
+  },
+];
```

Comment:
- [ ] You should remove all the hardcoded  array of items because now you can add and remove